.ci/packer_cache.sh

```diff
@@ -13,8 +13,8 @@ docker pull "$(make -C .ci --no-print-directory print-ci-image)"
 
 # Kind images (https://hub.docker.com/r/kindest/node/tags)
 # Pull exact images due to image incompatibility between kind 0.8 and 0.9 see https://github.com/kubernetes-sigs/kind/releases
```

Comment:
Not sure it is still needed...

Reply:
Do you mean the comment, the use of exact images or the whole script?

Use exact images? I think we still need it. I see 2 different `v1.23.6` images for kind 0.13.0 and 0.14.0.
The comment? Still good to get some context why we use the ids in the image names.
The script? I think it's worth it, it speeds up the job a bit.

I would just update the comment to make it more generic:

```suggestion
# Pull exact images due to images incompatibility between kind versions see https://github.com/kubernetes-sigs/kind/releases
```

Reply:
> Use exact images? I think we still need it. I see 2 different v1.23.6 images for kind 0.13.0 and 0.14.0.

The whole script, I don't think we need to use 2 different versions of Kind anymore. I feel it increases a bit the maintenance complexity for what seems to be  more or less 30 seconds from GCP:

```
time docker pull kindest/node:v1.24.1
v1.24.1: Pulling from kindest/node
f500296146dd: Pull complete
3db0d32e06a6: Pull complete
Digest: sha256:11276ce4763d5600379c0104c4cb51bf7420c8c02937708e958ad68d08e0910c
Status: Downloaded newer image for kindest/node:v1.24.1
docker.io/kindest/node:v1.24.1

real	0m30.683s
user	0m0.073s
sys	0m0.063s
```

That being said it's not a strong feeling 😄 ,  I'm perfectly fine with keeping it. Let's just update the comment.

Reply:
I'm ok to delete it and see if we really perceive an increase in the average duration of execution of the pr job.

Reply:
The fewer bits on the network, the better for the :earth_africa:?